superset-frontend/src/components/FlashError.jsx

```diff
@@ -0,0 +1,53 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { t } from '@superset-ui/translation';
+import withToasts from 'src/messageToasts/enhancers/withToasts';
+
+const propTypes = {
+  children: PropTypes.node.isRequired,
+  common: PropTypes.object.isRequired,
+};
+
+const flashObj = {
+  info: 'addInfoToast',
+  danger: 'addDangerToast',
+  warning: 'addWarningToast',
+  success: 'addSuccessToast',
+};
+
+class FlashError extends React.PureComponent {
+  componentDidMount() {
+    const flashArr = this.props.common.flash_messages;
+    if (flashArr.length > 0) {
+      flashArr.forEach((e, i) => {
+        const type = flashArr[i][0];
+        const text = flashArr[i][1];
+        const flash = flashObj[type];
+        this.props[flash](t(text));
```

Comment:
oh also, wrapping the text in `t()` won't do anything here since it's not a string to be translated. you can remove the `t`